docs/source/protocol.rst

```diff
@@ -30,28 +30,29 @@ Network Send Number
 - Begins at 0 on each client and goes up by 1 each time we attempt a metrics network upload.
 - Does not go up on network retries.
 
-Relative Timestamp
```

Comment:
I'm still a little unsure about removing the magic timestamp algorithm…

Reply:
Let’s keep it.